tests/lib/Validator/SpecialCharactersValidatorTest.php

```diff
@@ -66,7 +66,8 @@ public function testValidate(string $password, bool $enforced, bool $valid) {
 			$this->expectException(HintException::class);
 		}
 
-		$this->assertTrue($this->validator->validate($password));
+		$this->assertTrue(true);
```

Comment:
Really? What if if fails?

Reply:
Well it either throws an exception or not.
The assert is there to ensure phpunit is quite about risky tests

Reply:
I was referring to a joke http://geek-and-poke.com/geekandpoke/2014/1/15/philosophising-geeks